x-pack/plugins/actions/server/builtin_action_types/servicenow/index.test.ts

```diff
@@ -169,8 +175,8 @@ describe('validateParams()', () => {
 
 describe('execute()', () => {
   beforeEach(() => {
-    handleCreateIncidentMock.mockReset();
-    handleUpdateIncidentMock.mockReset();
+    handleIncidentMock.mockReset();
+    handleIncidentMock.mockReset();
```

Comment:
just once should do. you could also call `jest.resetAllMocks()` if theres more than one to reset